### PR TITLE
nvme-print: added missing fguid to json id-ctrl output

### PR DESCRIPTION
**Pull request by skasero**
- add fguid to json
- removed some whitespace

### CODE REVIEW

**Comment by igaw** [comment]
Hmm, I think this is colliding with https://github.com/linux-nvme/nvme-cli/pull/1652

You might want to coordinate the work on fguid.

**Review by pierrelabat — CHANGES_REQUESTED** [review]

**Review by skasero — COMMENTED** [review]
The error I got with uuid_string_t:
`error: ‘uuid_string_t’ undeclared (first use in this function)`

File: nvme-print.c [hunk 1]
```diff
@@ -251,8 +261,7 @@ static void json_nvme_id_ctrl(struct nvme_id_ctrl *ctrl,
 	long double maxdna = int128_to_double(ctrl->maxdna);
 
 	char sn[sizeof(ctrl->sn) + 1], mn[sizeof(ctrl->mn) + 1],
-		fr[sizeof(ctrl->fr) + 1], subnqn[sizeof(ctrl->subnqn) + 1],
-		fguid[sizeof(ctrl->fguid) + 1];
+		fr[sizeof(ctrl->fr) + 1], subnqn[sizeof(ctrl->subnqn) + 1], fguid[37];
```
[the remark this answers is not in the document]
Comment (igaw) [reply nvme-print.c:264]: Please also add a define for the magic number. BTW, isn't this missing the NULL char?

**Comment by igaw** [comment]
And merge these two patches into one while at it. Don't want to merge the development history. Thanks.

**Comment by igaw** [comment]
Should be fixed with https://github.com/linux-nvme/nvme-cli/commit/78b7ad235507ddd59c75c7fcc74fc6c927811f87  and https://github.com/linux-nvme/nvme-cli/commit/5acfc7215917f94d31fd05be7958114d7b3a85d6. I completely forgot about this PR when reviewing/working on #1652. Didn't want to steal your commit. Sorry.

**Pull request closed without merging**